tools/utils/gh.py

```diff
@@ -40,15 +40,15 @@ def download_onedir_artifact(
             )
             return ExitCode.FAIL
     else:
-        if arch == "arm64":
-            ctx.info(f"Turning passed arch {arch!r} into 'aarch64'")
-            arch = "aarch64"
+        if arch == "aarch64":
```

Comment:
Surely this requires some discovery since Redhat family it is aarch64, Debian family arm64

Reply:
We don't care about `aarch64` in this context.